data/README.txt

```diff
@@ -0,0 +1,3 @@
+Use this command from DogShow/ to import the data from puppy_seed.json
+
+mongoimport --db dogshow --collection dogs --drop --file data/puppy_seed.json --type json --jsonArray

```

Comment:
Perhaps there's a way to do it after "npm i" so that it only seeds when we install